internal/compiler/tests/syntax/elements/component_container.slint

```diff
@@ -3,25 +3,26 @@
 
 export component Foo inherits Rectangle {
     ComponentContainer {
-//  ^error{ComponentContainers may not have children}
+//  ^error{ComponentContainers disabled in compiler configuration}
         @children
     }
 }
 
 export Bar := Rectangle {
     Foo { Text {} }
     ComponentContainer {
-//  ^error{ComponentContainers may not have children}
+//  ^error{ComponentContainers disabled in compiler configuration}
         Text {}
     }
     ComponentContainer {
-//  ^error{ComponentContainers may not have children}
```

Comment:
I think we should keep both error.

Reply:
We don't have to keep both error.  But it would be nice to keep testing for this error, so enabling this special configuration in syntax_test.